py-polars/polars/internals/construction.py

```diff
@@ -32,33 +32,30 @@
 )
 from polars.utils import threadpool_size
 
+if TYPE_CHECKING:  # pragma: no cover
+    import pandas as pd
+
 try:
     from polars.polars import PyDataFrame, PySeries
 
     _DOCUMENTING = False
 except ImportError:  # pragma: no cover
     _DOCUMENTING = True
 
-if TYPE_CHECKING:  # pragma: no cover
```

Comment:
So this code should be used above.

Reply:
Yes, I understand what you mean. Let's keep to the thread above for this discussion 👍